mllib/src/test/scala/org/apache/spark/mllib/tree/EnsembleTestHelper.scala

```diff
@@ -70,7 +70,7 @@ object EnsembleTestHelper {
       metricName: String = "mse") {
     val predictions = input.map(x => model.predict(x.features))
     val errors = predictions.zip(input.map(_.label)).map { case (prediction, label) =>
-      prediction - label
+      label - prediction
```

Comment:
Yes, I suggested changing these purely for consistency of the convention. In several instances it doesn't actually matter to the result.